api/metrics/src/main/java/io/opentelemetry/api/metrics/Counter.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.metrics;
+
+/**
+ * A counter instrument.
+ *
+ * <p>This will have a specific subclass that recoords appropriate primitive values, e.g. {@link
```

Comment:
```suggestion
 * <p>Type-specific subclasses record appropriate primitive values, e.g. {@link
```

Present tense instead of future since it already does